fe/src/main/java/org/apache/doris/analysis/BrokerDesc.java

```diff
@@ -17,61 +17,36 @@
 
 package org.apache.doris.analysis;
 
-import org.apache.doris.common.io.Text;
-import org.apache.doris.common.io.Writable;
+import org.apache.doris.common.AnalysisException;
+import org.apache.doris.load.EtlJobType;
 import org.apache.doris.common.util.PrintableMap;
 
-import com.google.common.collect.Maps;
-
 import java.io.DataInput;
-import java.io.DataOutput;
 import java.io.IOException;
 import java.util.Map;
 
 // Broker descriptor
-public class BrokerDesc implements Writable {
-    private String name;
-    private Map<String, String> properties;
-
+//
+// Broker example:
+// WITH BROKER "broker0"
+// (
+//   "username" = "user0",
+//   "password" = "password0"
+// )
+public class BrokerDesc extends ResourceDesc {
```

Comment:
This is not a good idea to make Broker to extends Resource. I think they are two things

Reply:
ok